pkg/cmd/pulumi/stack_init_test.go

```diff
@@ -0,0 +1,68 @@
+package main
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/pulumi/pulumi/pkg/v3/backend"
+	"github.com/stretchr/testify/assert"
+)
+
+// This test demonstrates that validateCreateStack will filter
+// out teams consisting exclusively of whitespace. NB: It's not intended
+// to fully validate the correctness of team names. For example, it doesn't
+// check for illegal punctuation, length, or other measures of correctness.
+// To keep the codebase DRY, we pass along team names as-is to the Service,
+// with the exception of trimming whitespace, and allow the Service to
+// validate them.
+func TestValidateCreateStackOpts(t *testing.T) {
```

Comment:
There should probably be another test function to validate that when ValidateCreateStackOpts is called with a backend that doesn't support teams, and a CreateStackOpts with a non-empty team, it logs a message/returns an error/whatever.

Reply:
Added!